packages/@lwc/engine-core/src/framework/hydration.ts

```diff
@@ -34,6 +41,12 @@ import {
 import { patchProps } from './modules/props';
 import { applyEventListeners } from './modules/events';
 
+const enum EnvNodeTypes {
+    ELEMENT = 1,
+    TEXT = 3,
+    COMMENT = 8,
+}
```

Comment:
Let's add a comment to indicate that these come from [Node](https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType), since it may not be obvious.